tests/virt_autotest/virt_utils.pm

```diff
@@ -545,4 +545,25 @@ sub generate_testcase_xml {
     $xml_writer->endTag('testcase');
 }
 
+#Get installed SLE release version and service spack
+#get_sles_release only accepts /etc/os-release as source file, which can be on virtual guest or another physical host
+#Or it can be placed anywhere as long as it can be reached somehow, its absolute file path on the guest or host
+#should be passed in as the second parameter $os_release_file, for example, "/etc/os-release"
+#If sles version on virtual guest or another physical host needs to be parsed, connection method and guest or host name,
+#fqdn or ip address should be passed in as the first parameter $go_to_target, for example, "ssh root\@name"
+#For usage on locahost, no parameters need to be given
+sub get_sles_release {
```

Comment:
I have one last suggestion, perhaps this is a function for getting the installed SLE release version and service pack I think it belongs to `lib/version_utils.pm` so everyone can easy find it and use it. Can we move it over there?

Reply:
Should be OK. Done

Reply:
Just remove it from `@EXPORT` at `tests/virt_autotest/virt_utils.pm` and I am fine 😉

Reply:
Yes. Done.

Reply:
Thank you